src/status_im/ui2/screens/chat/actions.cljs

```diff
@@ -52,66 +58,199 @@
          (if muted?
            #(unmute-chat-action chat-id)
            #(mute-chat-action chat-id))
-         false))
+         false
+         nil
+         true))
 
 (defn mark-as-read-entry [chat-id]
-  (entry :main-icons2/check
+  (entry :main-icons2/correct
          (i18n/label :mark-as-read)
          #(mark-all-read-action chat-id)
+         false
+         nil
          false))
 
 (defn clear-history-entry [chat-id]
   (entry :main-icons2/delete
          (i18n/label :clear-history)
          #(clear-history-action chat-id)
-         true))
+         true
+         nil
+         false))
 
 (defn delete-chat-entry [chat-id]
   (entry :main-icons2/delete
          (i18n/label :delete-chat)
          #(delete-chat-action chat-id)
-         true))
+         true
+         nil
+         false))
+
+(defn leave-group-entry [chat-id]
+  (entry :main-icons2/log-out
+         (i18n/label :leave-group)
+         #(leave-group-action chat-id)
+         true
+         nil
+         false))
 
 (defn view-profile-entry [chat-id]
   (entry :main-icons2/friend
          (i18n/label :view-profile)
          #(show-profile-action chat-id)
+         false
+         nil
          false))
 
 (defn edit-nickname-entry [chat-id]
   (entry :main-icons2/edit
          (i18n/label :edit-nickname)
          #(edit-nickname-action chat-id)
+         false
+         nil
+         false))
+
+(defn notifications-entry []
+  (entry :main-icons2/notifications
+         (i18n/label :notifications)
+         #(js/alert "TODO: to be implemented, requires design input")
+         false
+         "All messages" ; placeholder
+         true))
+
+(defn fetch-messages-entry []
+  (entry :main-icons2/save
+         (i18n/label :fetch-messages)
+         #(js/alert "TODO: to be implemented, requires design input")
+         false
+         nil
+         true))
+
+(defn pinned-messages-entry []
+  (entry :main-icons2/pin
+         (i18n/label :pinned-messages)
```

Comment:
There are a couple translations missing the `:t/` qualification.

Reply:
I saw they were not used in this file, so I did so. Curious about what difference it makes?

Reply:
Thanks for asking @OmarBasem.

It's the same reasoning as to why we use `:i/<name-of-the-icon>`. It helps create a valid list of translation keywords in the codebase. It also helps to precisely grep for translations, which is sometimes important because some of them use very common words, like `admin`. Editors can easily autocomplete only translation keywords once you type `:t/`. In the future I bet we'll have a linter rule that if the function `i18n/label` is used, then the first argument must be qualified with `:t`.

The complete reasoning behind this convention is not in our docs, otherwise it would become too big if we explain everything in detail.

Reply:
Aha okay, thanks for the explanation. So the Clojure language ignores these qualifications? (Initially, I was expecting it not to work without the qualification)

Reply:
It all depends on what we do with the namespaced keyword. For example, if the function `i18n/label` had a `:pre` condition check that verified the namespace was indeed `:t`, then you'd get runtime errors signaling this problem. But if, on the other hand, the i18n/label simply uses the core `name` function, then the namespace is discarded (which is exactly what our codebase does).